src/validation.cpp

```diff
@@ -982,7 +982,6 @@ bool MemPoolAccept::Finalize(const ATMPArgs& args, Workspace& ws)
     const CAmount& nModifiedFees = ws.m_modified_fees;
     const CAmount& nConflictingFees = ws.m_conflicting_fees;
     const size_t& nConflictingSize = ws.m_conflicting_size;
-    const bool fReplacementTransaction = ws.m_replacement_transaction;
```

Comment:
Can also remove `m_replacement_transaction`, now that it is unused and inline `setConflicts.size()` again?

Reply:
Done.